README.md

```diff
@@ -5,84 +5,120 @@ Sawtooth, allowing users to exchange quantities of customized "Assets" with
 other users on the blockchain. This repo contains a number of components which
 together with a _Hyperledger Sawtooth_ validator, will run a Sawtooth
 blockchain and provide a simple RESTful API to interact with it. The components
-in this repo will include:
+in this repo include:
 
-- a **rest api** which provides a REST API for querying blockchain data
-- a **transaction processor** which handles RBAC-specific transaction logic
+- a **rest api** which provides HTTP/JSON endpoints for querying blockchain data
+- a **transaction processor** which handles Marketplace transaction logic
 - a **ledger sync** which writes blockchain state changes to a local database
-- and an example **client** which uses Marketplace to create a loyalty point
-  program administered through a simple web app.
+- **SawbuckManager** and an example client which uses Marketplace to create a
+  loyalty point program that users can interact with through a simple web app
+- a **shell** for with all of the dependencies necessary to run support scripts
 
 ## Usage
 
-The easiest way to run these components is with
-[Docker](https://www.docker.com/what-docker). To start these components,
-first install Docker for your platform and clone this repo.
+This project utilizes [Docker](https://www.docker.com/what-docker) to simplify
+dependencies and deployment. To start the various Marketplace components,
+first install and run Docker for your platform, then clone this repo.
```

Comment:
Does 'and run Docker' mean start the docker service? or test the installation?

Reply:
It means the Docker application/service. Whatever term we want to use for it. It is not referring to any particular commands.

Reply:
Suggestion: "and make sure the Docker service is started on your platform"